cypress/support/testData.js

```diff
@@ -0,0 +1,5 @@
+export const user = {
+    userName: 'B@ry!',

```

Comment:
Consider using a more descriptive username for testing purposes, such as 'TestUser' or 'CypressUser', to improve code readability 📖